src/pages/testnet/DevelopersPage.vue

```diff
@@ -12,137 +15,295 @@ export default {
             }
+        },
+        async onFaucet() {
+            this.submitting = true;
+            const result = await this.faucet(this.form.send_to);
+            this.handleAnswer(result, 'TLOS sent successfully');
+            this.submitting = false;
+        },
+        async onEvmFaucet() {
+            this.submitting = true;
+            const result = await this.evmFaucet(this.form.send_to_evm);
+            this.handleAnswer(result, 'TLOS sent successfully');
             this.submitting = false;
         },
         async onAccount() {
             this.submitting = true;
             const result = await this.account(this.form);
-            if (result) {
-                Notify.create({
-                    message: result,
-                    position: 'top',
-                    color: 'primary',
-                    textColor: 'white',
-                    actions: [{ label: 'Dismiss', color: 'white' }],
-                });
-                this.transactionId = result.transactionId;
-            }
+            this.handleAnswer(result, 'Account created successfully');
             this.submitting = false;
         },
+        checkTabFromUrl() {
+            const tabQueryParam = this.$route.query.tab;
+            if (tabQueryParam && available_tabs.includes(tabQueryParam)) {
+                this.tab = tabQueryParam;
+            } else {
+                this.tab = default_tab;
+            }
+        },
+    },
+    watch: {
+        '$route.query.tab'(newTab) {
+            this.tab = newTab || default_tab;
+        },
+        tab(newTab) {
+            this.$router.push({ query: { tab: newTab }});
+        }
     },
 };
 </script>
 
-<template lang="pug">
-q-page.flex.flex-center
-  q-card
-    q-card-section
-      div.center.text-h4 Testnet faucet
-    q-card-section
-      q-input.q-mb-lg(
-        ref="account_name"
-        v-model="form.account_name"
-        color="accent"
-        label="Account name"
-        outlined
-        maxlength="12"
-      )
-      q-input.q-mb-lg(
-        ref="owner_key"
-        v-model="form.owner_key"
-        color="accent"
-        label="Owner key"
-        outlined
-      )
-      q-input.q-mb-lg(
-        ref="active_key"
-        v-model="form.active_key"
-        color="accent"
-        label="Active key"
-        outlined
-      )
-      q-btn.full-width(
-        color="primary"
-        label="Create testnet account"
-        size="lg"
-        unelevated
-        :loading="submitting"
-        @click="onAccount"
-      )
+<template>
+    <q-page class="p-dev-page flex flex-center">
+        <div class="p-dev-page__title-card">
+            <div class="p-dev-page__title">Testnet Developers Tools</div>
+            <div class="text-subtitle2">
+                Use the following tools to create a new account,
+                send TLOS tokens to a Telos native account,
+                or send TLOS tokens to an EVM compatible address.
+            </div>
```

Comment:
Julio, please help me write something intelligent here to welcome new developers.